e2e/test/scenarios/metrics/reproductions/metrics-reproductions.cy.spec.ts

```diff
@@ -61,3 +70,81 @@ describe("issue 47058", () => {
     });
   });
 });
+
+describe("Issue 44171", () => {
+  const METRIC_A: StructuredQuestionDetails = {
+    name: "Metric 44171-A",
+    type: "metric",
+    display: "line",
+    query: {
+      "source-table": ORDERS_ID,
+      aggregation: [["count"]],
+      breakout: [
+        [
+          "field",
+          ORDERS.CREATED_AT,
+          { "temporal-unit": "month", "base-type": "type/DateTime" },
+        ],
+      ],
+    },
+  };
+
+  const METRIC_B: StructuredQuestionDetails = {
+    name: "Metric 44171-B",
+    type: "metric",
+    display: "line",
+    query: {
+      "source-table": ORDERS_ID,
+      aggregation: [["count"]],
+      breakout: [
+        [
+          "field",
+          ORDERS.CREATED_AT,
+          { "temporal-unit": "month", "base-type": "type/DateTime" },
+        ],
+      ],
+    },
+  };
+
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+
+    createQuestion(METRIC_A);
```

Comment:
I'd put second createQuestion in `.then` to be sure that first was created

Reply:
No need, Cypress will wait for it